merlin/models/tf/utils/testing_utils.py

```diff
@@ -76,7 +76,7 @@ def assert_model_is_retrainable(
 
 def model_test(
     model: Model,
-    dataset: Union[merlin.io.Dataset, Loader],
+    dataloader: Union[merlin.io.Dataset, Loader],
```

Comment:
maybe a name like `dataset_or_loader` would help make this clear what the value is expected to be?